backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -2199,6 +2205,125 @@ describe('UserResolver', () => {
       })
     })
   })
+
+  describe('user', () => {
+    beforeEach(() => {
+      jest.clearAllMocks()
+    })
+
+    describe('unauthenticated', () => {
+      it('throws and logs "401 Unauthorized" error', async () => {
+        await expect(
+          query({
+            query: userQuery,
+            variables: {
+              identifier: 'identifier',
+            },
+          }),
+        ).resolves.toEqual(
+          expect.objectContaining({
+            errors: [new GraphQLError('401 Unauthorized')],
+          }),
+        )
+        expect(logger.error).toBeCalledWith('401 Unauthorized')
+      })
+    })
+
+    describe('authenticated', () => {
+      beforeAll(async () => {
+        user = await userFactory(testEnv, bibiBloxberg)
+        await mutate({
+          mutation: login,
+          variables: { email: 'bibi@bloxberg.de', password: 'Aa12345_' },
+        })
+      })
+
+      describe('identifier is no gradido ID and no email', () => {
+        it('throws and logs "Unknown identifier type" error', async () => {
+          await expect(
+            query({
+              query: userQuery,
+              variables: {
+                identifier: 'identifier',
+              },
+            }),
+          ).resolves.toEqual(
+            expect.objectContaining({
+              errors: [new GraphQLError('Unknown identifier type')],
+            }),
+          )
+          expect(logger.error).toBeCalledWith('Unknown identifier type', 'identifier')
+        })
+      })
+
+      describe('identifier is not found', () => {
+        it('throws and logs "No user found to given identifier" error', async () => {
+          await expect(
+            query({
+              query: userQuery,
+              variables: {
+                identifier: '00000000-0000-0000-0000-000000000000',
+              },
+            }),
+          ).resolves.toEqual(
+            expect.objectContaining({
+              errors: [new GraphQLError('No user found to given identifier')],
+            }),
+          )
+          expect(logger.error).toBeCalledWith(
+            'No user found to given identifier',
+            '00000000-0000-0000-0000-000000000000',
```

Comment:
please see my comments in backend/src/graphql/resolver/util/findUserByIdentifier.ts how to check and verify a correct uuid of version-4.
Then this testdata of a uuid will fail.